rest-service/manager_rest/test/endpoints/test_plugins_updates.py

```diff
@@ -177,16 +200,14 @@ def test_raises_with_real_active_plugins_updates_and_forced(self):
                 'update IDs: {0}'.format(plugins_update.id)):
             self.client.plugins_update.update_plugins('hello_world', True)
 
-    def test_raises_when_no_plugin_change_detected(self):
+    def test_no_op_when_no_plugin_change_detected(self):
```

Comment:
add another test for two no_op requests

Reply:
Though this is necessary IMO, one test for "no deployments to update" and one for "no plugins to update" - two different scenarios.

Reply:
`test_no_op_blueprint_has_no_deployments` is the "no deployments to update" scenario.

Reply:
I mean try to plugin update twice when every time you will get nothing todo

Reply:
The plugin update execution doesn't even execute. I've added a state check to the integration tests.